frontend/packages/ux-editor/src/containers/FormItemContext.test.tsx

```diff
@@ -1,53 +1,41 @@
-import React from 'react';
+import React, { act } from 'react';
```

Comment:
Don't think we should use `act` anymore 😄

Reply:
Done